CHANGELOG.md

```diff
@@ -1,24 +1,32 @@
 # Changelog
+
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+## [1.2.0 alpha 1] - March 4th, 2020
+
+### New Features
+
+- Updated minor version of core `@optimizely/optimizely-sdk` dependency
```

Comment:
Let's mention the effect of this: the change in the core SDK which will impact the log levels in this SDK too.